Formula/ibex.rb

```diff
@@ -12,37 +22,59 @@ class Ibex < Formula
     sha256 "41327b6a0da9b8b2ed888c3c2c33d5f6ff061ea13eceb4b9e8b6e660049d624d" => :mountain_lion
   end
 
-  option "with-java", "Build Java bindings for Choco solver."
+  option "with-java", "Enable Java bindings for CHOCO solver."
+  option "with-ampl", "Use AMPL file loader plugin"
+
+  option "without-ensta-robotics", "Don't build the Contractors for robotics (SLAM) plugin"
+  option "without-param-estim", "Don't build the Parameter Estimation (enhanced Q-intersection algorithm) plugin"
```

Comment:
These optional plugins are supported by different people than the main Ibex library. They are not enabled in Ibex CI system. Thus we may want to disable these when building head.